app/config/index.js

```diff
@@ -2,10 +2,11 @@ import local from './local'
 import development from './development'
 import production from './production'
 import colors from './colors'
+import store from './store'
 
 let config = {
   colors,
-  storybookEnabled: true,
+  // storybookEnabled: true,
```

Comment:
Remove this.

Reply:
your `local.js` should look like:

```
export default {
  storybookEnabled: true
}
```
It will not be commuted.

Then you should add a `local-example.js` (which will be committed) which an example of how it should look so your collaborators can see what their local config should be like.

Reply:
storybookEnabled removed from index.js.
Created local-example.js and 
export default {
  storybookEnabled: false
}
for an example. Can you please suggest we should add few more example with default value which we are using as local variable.

Reply:
Add config values as you need them. 

It could be things like:
-  the apiHost - locally you want to point to localhost/dev server
- enabled/disabling analytics - you don't want development analytics to be tracked in the production analytics dashboard

In the context of this app I don't think you have any other config options at the moment.

Reply:
Yes, We do not have any other config rather than storybookEnabled now. If we require any other config we will add.

Reply:
Please review if anything is pending at my side.